lib/rsr_group/version.rb

```diff
@@ -1,3 +1,3 @@
 module RsrGroup
-  VERSION = '5.0.3'.freeze
+  VERSION = '5.0.2'.freeze
```

Comment:
You may need to increment to `5.0.4` to re-release, no?

Reply:
Yea, I think you're right